<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>123</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="CalibrateVoltages">
        <property name="text">
         <string>Calibrate Voltages</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="CalibrateDetector">
        <property name="text">
         <string>Calibrate Detector</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="ControlSupplies">
        <property name="text">
         <string>Control Supplies</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="SupplyInfo">
        <property name="text">
         <string>Supply Info</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="DetectorInfo">
        <property name="text">
         <string>Detector Info</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>123</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
